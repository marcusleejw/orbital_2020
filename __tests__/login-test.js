import 'react-native';
import React from 'react';
import Cart from '../userScreens/cart';
import renderer from 'react-test-renderer';

test('User Login snapshot', () => {
  const snap = renderer.create(
    <Cart />
  ).toJSON();

  expect(snap).toMatchSnapshot();
}); 