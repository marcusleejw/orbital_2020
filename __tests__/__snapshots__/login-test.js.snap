// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Login snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "display": "flex",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "padding": 35,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#f01d71",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 10,
        "textAlign": "center",
      }
    }
  >
    Please make a selection:
  </Text>
  <View
    style={
      Object {
        "flexDirection": "column",
        "justifyContent": "space-between",
        "paddingTop": 10,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#f01d71",
            "borderRadius": 8,
            "marginVertical": 10,
            "paddingHorizontal": 10,
            "paddingVertical": 30,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 16,
              "textAlign": "center",
              "textTransform": "uppercase",
            }
          }
        >
          Order Food
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#f01d71",
            "borderRadius": 8,
            "marginVertical": 10,
            "paddingHorizontal": 10,
            "paddingVertical": 30,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 16,
              "textAlign": "center",
              "textTransform": "uppercase",
            }
          }
        >
          Check Queue Number
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#f01d71",
            "borderRadius": 8,
            "marginVertical": 10,
            "paddingHorizontal": 10,
            "paddingVertical": 30,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 16,
              "textAlign": "center",
              "textTransform": "uppercase",
            }
          }
        >
          Log Out
        </Text>
      </View>
    </View>
  </View>
</View>
`;
